### DISCUSSION TITLE
Remove autogenerated notes from synced tasks?

### DISCUSSION THREAD

**Discussion by tjex** [bergercookie/syncall#138] [Q&A]
Hello. When syncing to caldav at least, I assume it does this for all services, syncall adds to each task:

```text
IMPORTED FROM TASKWARRIOR

*uuid: <uuid>
```

This makes my reminders app on the phone very messy with info that I don't want.. Is there a way to remove this?

I'm fearing that the uuid field is required for syncall to function correctly..? But hopefully at least the all caps phrase can be removed.

**Comment by mrwsl** [comment]
Did you find a solution for this?

**Reply by tjex** [reply]
no, did not :(
